libs/market-list/src/lib/market-list-table.tsx

```diff
@@ -1,6 +1,6 @@
 import { forwardRef } from 'react';
 import type { ValueFormatterParams } from 'ag-grid-community';
-import { PriceCell, formatNumber, t } from '@vegaprotocol/react-helpers';
```

Comment:
PriceCell was renamed to PriceFlashCell, and PriceCell was added which formats price but don't flash on change.